src/components/hacks/aurora/stats.tsx

```diff
@@ -0,0 +1,46 @@
+import Image from "next/image";
+
+const Stats = () => {
+  return (
+    <div className="flex flex-col justify-center py-30">
+      <div className="text-acm-gray-500 grid translate-x-[15%] [grid-template-columns:280px_250px_400px] gap-10 pb-18">
```

Comment:
```suggestion
      <div className="text-acm-gray-500 grid translate-x-[15%]  gap-10 pb-18">
```
this is unoptimal. instead have two grids since they seem to have different widths. these grids can have differing width lengths but the idea is you have margin left instead of translate-x